style/preview.less

```diff
@@ -215,6 +215,55 @@
 		}
 	}
 
+	&-error {
+		height: 285px;
+
+		&-body {
+			background-color: #f8f9fa;
+			height: 100%;
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+
+			&-message {
+				display: flex;
+				margin-left: 8%;
+				margin-right: 8%;
+				font-family: 'Lato', sans-serif;
```

Comment:
we don't set font-family in the entire library, we use default font for all.

Reply:
Do you mean I should remove the `font-family` lines?